### PR TITLE
Update 1password extension

### LINKED ISSUE #13513: [1Password] Item does not contain the field 'one-time password'.
### Extension

https://www.raycast.com/khasbilegt/1password

### Raycast Version

1.78.1

### macOS Version

_No response_

### Description

I use raycast to load the 1password plugin

After configuring 1password-CLI, it works well and can read the account and password I specified normally.

But there is a problem when reading the one-time password.

I clearly set a one-time password in 1password, but I can't access it through raycast, and it keeps prompting that the one-time password cannot be found

> Item does not contain the field 'one-time password'.

Of course, one-time passwords definitely exist in 1password

After further investigation, I found that my 1password is a Chinese version, so the key corresponding to the one-time password in the cli is [一次性密码], not [one-time password]

So of course it cannot be accessed.

So for Chinese users, can we optimize it and choose the language for extracting the one-time password according to the version of 1password instead of always using [one-time password]

### Steps To Reproduce

As above

### Current Behaviour

_No response_

### Expected Behaviour

_No response_

### PR DESCRIPTION
**Pull request by bxb100**
## Description

- close #13513
- add new view

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/1password/src/generate-password.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { getPreferenceValues, PreferenceValues } from "@raycast/api";
+import { AuthProvider } from "./v8/components/AuthContext";
+import { RandomPassword } from "./v8/components/RandomPassword";
+
+export default function Command() {
+  return getPreferenceValues<PreferenceValues>().version == "v8" ? (
```
Comment (LitoMore) [inline extensions/1password/src/generate-password.tsx:6]:
```suggestion
  return getPreferenceValues<PreferenceValues>().version === "v8" ? (
```

Let's use strict equal.

File: extensions/1password/src/v8/components/RandomPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              concealed={true}
+              content={generatedPassword || ""}
+              icon={Icon.Clipboard}
+              shortcut={Shortcut.Common.Copy}
+            />
+          )}
+        </ActionPanel>
+      }
+    >
+      <Form.Description title={"🔑Password"} text={`${generatedPassword || "Generating..."}\n\n\n`} />
```
Comment (LitoMore) [inline extensions/1password/src/v8/components/RandomPassword.tsx:104]:
```suggestion
      <Form.Description title={"🔑 Password"} text={`${generatedPassword || "Generating..."}\n\n\n`} />
```

Let's insert a space between them for a better visual.

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by bxb100** [comment]
@LitoMore Thanks review, all settle

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**